exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -172,19 +170,30 @@ func (b *BTCMarkets) Start(wg *sync.WaitGroup) {
 // Run implements the BTC Markets wrapper
 func (b *BTCMarkets) Run() {
 	if b.Verbose {
+		log.Debugf(log.ExchangeSys,
+			"%s Websocket: %s (url: %s).\n",
+			b.Name,
+			common.IsEnabled(b.Websocket.IsEnabled()),
+			btcMarketsWSURL)
 		b.PrintEnabledPairs()
 	}
 
-	forceUpdate := false
-	if !common.StringDataContains(b.GetEnabledPairs(asset.Spot).Strings(), "-") ||
-		!common.StringDataContains(b.GetAvailablePairs(asset.Spot).Strings(), "-") {
-		enabledPairs := []string{"BTC-AUD"}
-		log.Warnln(log.ExchangeSys, "Available pairs for BTC Markets reset due to config upgrade, please enable the pairs you would like again.")
-		forceUpdate = true
+	var forceUpdate bool
+	if forceUpdate {
```

Comment:
This will never execute, please preserve the original condition (checking to see if there's no delimiter on the enabled pairs or enabled pairs) and also change the currency.Pair to BTC-AUD as USDT isn't supported on BTC Markets